ui/app/components/ui/list-item/index.scss

```diff
@@ -11,19 +11,28 @@
 
   display: flex;
   justify-content: flex-start;
-  align-items: stretch;
+  align-items: center;
 
-  &__icon > * {
-    margin: 8px 14px 0 0;
+  &__icon {
+    display: flex;
+    align-items: center;
+    > * {
+      margin: 0 16px 0 0;
+    }
```

Comment:
Nit: I'm not sure these rules make a difference either

```suggestion
  &__icon > * {
    margin: 0 16px 0 0;
  }
```

Reply:
Those rules are how the icon is being centered.
![icon](https://user-images.githubusercontent.com/4448075/83904536-a3165600-a725-11ea-8c2f-3ec9dcfeaee5.gif)